mime-ballerina/Module.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+
+This module provides APIs to encapsulate multiple body parts such as attachments into a single message. 
+
+The communication of such messages follow the MIME (Multipurpose Internet Mail Extensions) specification as specified in 
+the [RFC 2045 standard](https://www.ietf.org/rfc/rfc2045.txt).
```

Comment:
I think we can remove this.

Reply:
Remove the linked rfc? Even GO has specified the rfc https://golang.org/pkg/mime/multipart/